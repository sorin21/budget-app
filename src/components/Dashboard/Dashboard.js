import React, { Component } from 'react';

import ExpensesList from '../ExpensesList/ExpensesList';
import ExpensesListFilters from '../ExpensesList/ExpensesListFilters/ExpensesListFilters';
import Hoc from '../../playground/hoc';

class Dashboard extends Component {
  render() {
    return (
      <div>
        Dashboard
        <ExpensesListFilters />
        <ExpensesList />
        <Hoc isAdmin={false} info="There are the details" />
      </div>
    );
  }
}

export default Dashboard;