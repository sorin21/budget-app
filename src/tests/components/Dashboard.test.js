import React from 'react';
import { shallow } from 'enzyme';

import Dashboard from '../../components/Dashboard/Dashboard';

test('should render Dashboard component', () => {
  const wrapper = shallow(<Dashboard />);
  expect(wrapper).toMatchSnapshot();
});