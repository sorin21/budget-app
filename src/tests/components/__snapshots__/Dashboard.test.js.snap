// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Dashboard component 1`] = `
<div>
  Dashboard
  <Connect(ExpensesListFilters) />
  <Connect(ExpensesList) />
  <Component
    info="There are the details"
    isAdmin={false}
  />
</div>
`;
